flight_computer/main.py

```diff
@@ -20,5 +22,12 @@ def radio_task():
 
         sleep(0.5)
 
+def altimeter_task():
+    altimeter = Altimeter()
```

Comment:
you didnt give a file_name here which means file_path is most likely = NULL. you could add a default file name in your Altimeter class to make this work!